<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrediteeForm</class>
 <widget class="QWidget" name="CrediteeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>118</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>390</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>390</width>
    <height>118</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Creditee Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#userFrame{
background-color:#2e8cff;
color:white;
font:14px;
padding:0px;
border-radius:20px;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="userFrame">
     <property name="styleSheet">
      <string notr="true">*{
background-color:#2e8cff;
}
QPushButton#userPicture{
border:1px solid white;
border-radius:50px;
}
QPushButton#deleteUserBtn{
background-color:red;
border-radius:19px;

}
QPushButton#editUserBtn{
border-radius:20px;
background-color:green;
}
QPushButton#deleteUserBtn:hover{
border:2px solid white;
}
QPushButton#editUserBtn:hover{
border:2px solid white;
}
QPushButton#deleteUserBtn:pressed{
color: rgba(255, 0, 0,190);
border:3px solid orange;
}
QPushButton#editUserBtn:pressed{
background-color: rgba(0, 128, 0,190);
border:3px solid orange;
}

QPushButton#repayBtn{
border-radius:10px;
background-color:transparent;
border:1px dotted grey;
}
QPushButton#repayBtn:hover{
border:2px solid white;
}
QPushButton#repayBtn:pressed{
border:3px solid orange;
}

QPushButton#cleardebtBtn{
border-radius:10px;
background-color:transparent;
border:1px dotted grey;
}
QPushButton#cleardebtBtn:hover{
border:2px solid white;
}
QPushButton#cleardebtBtn:pressed{
border:3px solid orange;
}

</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="userPicture">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>70</width>
          <height>70</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(85, 0, 127);</string>
            </property>
            <property name="text">
             <string>Full Name :</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="fullNameLabel">
            <property name="text">
             <string>My Name</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(85, 0, 127);</string>
            </property>
            <property name="text">
             <string>Mobile No :</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="mobileNoLabel">
            <property name="text">
             <string>+254700000000</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(85, 0, 127);</string>
            </property>
            <property name="text">
             <string>Debt Balance : </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="roleLabel">
            <property name="text">
             <string>0.00</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="repayBtn">
            <property name="minimumSize">
             <size>
              <width>45</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>45</width>
              <height>40</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Repay Debt</string>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 127, 186);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cleardebtBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>45</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>45</width>
              <height>40</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Waive All Debt</string>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 127, 186);</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="editUserBtn">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Edit Creditee Details</string>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 184, 89);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteUserBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Delete Creditee Account and Debt</string>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(188, 62, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
